cardinal/world_context.go

```diff
@@ -0,0 +1,42 @@
+package cardinal
+
+import (
+	"github.com/rs/zerolog"
+	"pkg.world.dev/world-engine/cardinal/ecs"
+)
+
+type WorldContext interface {
+	NewSearch(filter CardinalFilter) (*Search, error)
+	getECSWorldContext() ECSWorldContext
+	CurrentTick() uint64
+	Logger() *zerolog.Logger
+	IsReadOnly() bool
```

Comment:
IsReadOnly should not be included in this interface.

Reply:
fine